packages/SwingSet/src/liveslots/liveslots.js

```diff
@@ -710,6 +723,7 @@ function build(
         // this is a new import value
         val = makeImportedPresence(slot, iface);
       } else if (type === 'promise') {
+        // XXX note: this assert is the same as the one 5 lines above and therefor pointless
```

Comment:
agreed, that's vestigal, we can remove it the next time we swing through this neighborhood